website/docs/reference/integrations/datadog.md

```diff
@@ -16,7 +16,6 @@ The Datadog integration will perform a single retry if the HTTP POST against the
 You can choose to trigger updates for the following events:
 
 - feature-created
-- feature-updated (*)
```

Comment:
Maybe we should keep this one in the docs while it's still present in the `datadog-definition`?